### PR TITLE
Updated Contribution Guidelines

### PR DESCRIPTION
**Pull request by marvinbuss**
## Summary of the Pull Request
This PR adds details about the steering committee: Members, Responsibilities and Contribution scope.

## References
None

## PR Checklist
* [x] CLA signed. If not, go over [here](https://cla.opensource.microsoft.com/microsoft/Terminal) and sign the CLA

## Validation Steps Performed
No Validation required.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ This project welcomes community contributions. The contributions could be [Issue
 
 ## Open Development Workflow
 
-The Customer Architecture Team (CAT) is very active in this GitHub repository and carries out all the development work in the public. New issues, ideas or feature requests are filed in the repo and fixes/new implementations are implemented through pull requests that are linked to their respective issues. Code reviews can be followed as well to learn about the principles which the team follows in building and maintaining artefacts related to this repository..
+The Customer Architecture Team (CAT) is very active in this GitHub repository and carries out all the development work in the public. New issues, ideas or feature requests are filed in the repo and fixes/new implementations are implemented through pull requests that are linked to their respective issues. Code reviews can be followed as well to learn about the principles which the team follows in building and maintaining artefacts related to this repository.
```
Comment (amanjeetsingh) [inline CONTRIBUTING.md:9]: Consider rephrasing? 

"The Customer Architecture Team (CAT) is very active in this GitHub repository and carries out all the development work in the public."

Change to ... 

"The Customer Architecture Team (CAT) actively manages and contributes to this repository. "

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -90,6 +90,40 @@ If you want the team to review the changes, mark the pull request as "Ready for
 
 After the pull request has been reviewed and approved, the feature branch will be merged into the main branch and automatically closed and deleted.
 
+## Steering Committee
+
+The Enterprise-Scale Analytics Committee and its members (aka Committee Members) are the primary caretakers of the Enterprise-Scale Analytics repos including language, design, policy oversight, and reference implementations.
+
+Voting members may be added once approved by the Steering Committee. All members abide by all Organization polices, including the code of conduct. Voting members may be added or removed by no less than 3/4 affirmative vote of the Steering Committee. The membership of a Steering Committee member can terminate in case of written notice of resignation or if the member is unreachable or unresponsive for more than three months.
```
Comment (amanjeetsingh) [inline CONTRIBUTING.md:97]: "Organization polices" change to "organizational policies"

**Review by amanjeetsingh — COMMENTED** [review]
Minor updates

File: CONTRIBUTING.md (hunk 2 above)
Comment (marvinbuss) [reply CONTRIBUTING.md:97]: Changed

File: CONTRIBUTING.md (hunk 1 above)
Comment (marvinbuss) [reply CONTRIBUTING.md:9]: Changed

**Comment by marvinbuss** [comment]
Thanks @amanjeetsingh for the review. Changed the sentences as requested.

**Review by uday31in — APPROVED** [review]

**Review by marvinbuss — COMMENTED** [review]
LGTM!

**Review by amanjeetsingh — APPROVED** [review]
Good to go

**Review by esbran — APPROVED** [review]
Looks good

**Pull request merged**